.envrc

```
@@ -29,23 +29,45 @@ export POSTGRES_PASSWORD="$(cat ./postgres.password)"
 
 export POSTGRES_DB="cexplorer"
 
-# We bring together the above variables into a database connection string:
-export DB_SYNC_POSTGRES_CONNECTION_STRING="psql://$POSTGRES_USER:$POSTGRES_PASSWORD@$POSTGRES_HOST:$POSTGRES_PORT/$POSTGRES_DB"
-
 
 # To start with debug logs, add "-l debug" to APPEND_ARGS
 # To expose safe rpc method to the host port 9944, add "--unsafe-rpc-external" to APPEND_ARGS or  --validator
 export APPEND_ARGS="--allow-private-ip --pool-limit 10 --trie-cache-size 0 --prometheus-external --rpc-external --rpc-cors all"
 
 
-# Validator Values:
-if [ ! -f node.privatekey ]; then
-  # generate node key like this:
-  DOCKER_DEFAULT_PLATFORM=linux/amd64 docker run --rm -it docker.io/parity/subkey:latest generate-node-key  | sed -n '2p' > midnight-node.privatekey
-  # Use the second line of output for NODE_KEY (that's what sed -n '2p' does)
+# Node Key:
+# Generate a unique node key for P2P networking
+if [ ! -f midnight-node.privatekey ]; then
+  # Generate node key using parity/subkey
+  DOCKER_DEFAULT_PLATFORM=linux/amd64 docker run --rm docker.io/parity/subkey:latest generate-node-key > midnight-node.privatekey
 fi
 export NODE_KEY="$(cat ./midnight-node.privatekey)"
 
+# Indexer Values:
+if [ ! -f indexer.secret ]; then
+  # Generate 32-byte hex secret for indexer
+  openssl rand -hex 32 > indexer.secret
```

Let's stick to uuidgen - it's available on more systems and we use it above:

```suggestion
  uuidgen | tr -d '-' | head -c 32 > indexer.secret
```